docs/reference/contributing/guidelines/workflow.md

```diff
@@ -43,44 +43,37 @@ We copy issues reported on GitHub to our internal tracker and regularly triage t
 
 Pull requests on GitHub have to meet the following requirements to keep the code and commit history clean:
 
-- Commits should always contain a proper description of their content. Start with a concise and sensible one-line description. Then, elaborate on reasoning of the choices made, descriptions for reviewers and other information that might otherwise be lost.
-- You should always write commits to allow publication, so they can never contain confidential information, reference private documents, links to intranet locations or rude language.
+- Commits must always contain a proper description of their content. Start with a concise and sensible one-line description. Then, elaborate on reasoning of the choices made, descriptions for reviewers and other information that might otherwise be lost.
+- You should always write commits that allow publication. They must not contain confidential information, references to private documents, links to intranet locations or rude language.
 - Each commit should be the minimum self-contained commit for a change. A commit should always result in a new state that is again in a compilable state. You should (if possible) split large changes into logical smaller commits that help reviewers follow the reasoning behind the full change.
-- Commits and pull request titles should follow [Chris Beam’s seven rules of great commit messages](http://chris.beams.io/posts/git-commit#seven-rules):
```

Comment:
Query: Why did we remove these rules?

Reply:
Why have them listed here if they are straight behind the link?